### PR TITLE
Feature: Add en-us as localized language

### PR DESCRIPTION
**Pull request by wutschel**
## Description

Based on a discussion on the forums. This PR add US localization for the English translations. Only current difference is _Favourites_ vs. _Favorites_.

## Summary for release notes

Feature: Add en-us as localized language

### CODE REVIEW

**Comment by gade01** [comment]
@wutschel Please ping if you merge this PR.
Then I'll add the language at Weblate.

**Comment by wutschel** [comment]
@kambala-decapitator, should we add this one? It is for now only changing a single word, but more might come.

**Comment by kambala-decapitator** [comment]
> @kambala-decapitator, should we add this one? It is for now only changing a single word, but more might come.

sure. why not

**Comment by wutschel** [comment]
> sure. why not

Reade for review now.

File: XBMC Remote/Settings.bundle/cs.lproj/Root.strings [hunk 1]
```diff
@@ -24,7 +24,7 @@
 "Show TV SHOWS" = "Show TV SHOWS";
 "Show PICTURES" = "Show PICTURES";
 "Show LIVE TV" = "Show LIVE TV";
-"Show FAVOURITES" = "Show FAVOURITES";
+"Show FAVOURITES" = "Show FAVORITES";
```
Comment (kambala-decapitator) [inline XBMC Remote/Settings.bundle/cs.lproj/Root.strings:27]: why is this change needed? That's Czech language, isn't it?

File: XBMC Remote/en-US.lproj/Localizable.strings [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/* 
+  Localisable.strings
+  XBMC Remote
+
+  Created by Andree Buschmann on 21/11/20.
+  Copyright (c) 2012 joethefox inc. All rights reserved.
```
Comment (kambala-decapitator) [inline XBMC Remote/en-US.lproj/Localizable.strings:6]: I think we should change Company Name to `Kodi` (also in Xcode project settings)

File: XBMC Remote/Settings.bundle/cs.lproj/Root.strings (hunk 1 above)
Comment (wutschel) [reply XBMC Remote/Settings.bundle/cs.lproj/Root.strings:27]: Yes, my fault. Will be removed.

File: XBMC Remote/en-US.lproj/Localizable.strings (hunk 2 above)
Comment (wutschel) [reply XBMC Remote/en-US.lproj/Localizable.strings]: Did for the file which I added. There are a lot of places where this is used. Shall this be done for all files? And, where exactly should I change Xcode project settings? I cannot find any "joethefox inc" in there.

Comment (kambala-decapitator) [reply XBMC Remote/en-US.lproj/Localizable.strings]: You can change only in files that you created, original ones shouldn't be touched I think.

To change company name, click on the project itself, then open the right sidebar and select the leftmost "tab" there.

Comment (wutschel) [reply XBMC Remote/en-US.lproj/Localizable.strings]: Done.

**Review by kambala-decapitator — APPROVED** [review]

**Comment by wutschel** [comment]
Squashed the changes.

@gade01, this will be merged now.

**Comment by gade01** [comment]
@wutschel Great, thanks.
I'll add the language at Weblate later.

**Comment by gade01** [comment]
@wutschel Thanks.
I'll look at everything tomorrow morning.
